README.md

```diff
@@ -1,20 +1,19 @@
 ## Introduction
 
-The GORpipe analysis tool is developed and released here by WuXi Nextcode Genomics. It originates from the pioneers of population based genomic analysis, deCODE genetics, headquartered in Reykjavik, Iceland.
+The GORpipe analysis tool is developed and released by WuXi NextCODE Genomics. It originates from the pioneers of population based genomic analysis, deCODE Genetics, headquartered in Reykjavik, Iceland.
```

Comment:
It looks like it's called "deCODE genetics" with lowercase: https://www.decode.com/

Reply:
fixed